lib/mastodon/redis_config.rb

```diff
@@ -20,6 +20,24 @@ def setup_redis_env_url(prefix = nil, defaults = true)
       end.normalize.to_str
     end
   end
+
+  return unless ENV["#{prefix}REDIS_SENTINEL"] || ENV['REDIS_SENTINEL']
```

Comment:
@oneiros I think this environment variable would be better called `REDIS_SENTINELS` instead, since it can/does contain multiple values.